internal/controlplane/handlers_githubwebhooks.go

```diff
@@ -272,6 +272,25 @@ func (s *Server) HandleGitHubWebHook() http.HandlerFunc {
 	}
 }
 
+// HandleGitHubMarketplaceWebhook handles incoming GitHub Marketplace webhooks
+func (s *Server) HandleGitHubMarketplaceWebhook() http.HandlerFunc {
```

Comment:
Can we just name this `no-op` handler? I'd really like to move away from doing github-specific random things.